src/main/scala/org/broadinstitute/dsde/workbench/avram/db/AvramPostgresProfile.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.dsde.workbench.avram.db
+
+import slick.jdbc.PostgresProfile
+import com.github.tminglei.slickpg._
+import slick.basic.Capability
+import slick.jdbc.JdbcCapabilities
+
+trait AvramPostgresProfile extends PostgresProfile with PgJsonSupport with PgSearchSupport with PgSprayJsonSupport {
```

Comment:
I don't thing we need PgSearchSupport, I'll test it without it, that was a copy paste leftover